docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Layered Time Series Line Chart with Gauge as Average Line](../images/workflows/custom-chart-layered-time-series-gauge-chart-result.png "Layered Time Series Line Chart with Gauge as Average Line")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
```

Comment:
(optional) I don't think this line is needed. But, I would do a : here instead of the ...

Reply:
I did it this way to format it the same way as the Data-Table Block docs http://127.0.0.1:8000/dashboards/data-table/